README.md

```diff
@@ -1,2 +1,36 @@
 # version-checker
-Package version checker action
+
+Check package version against the base branch in pull requests.
+Fails the build if package version the same or older.
+
+Based on kriasoft/check-version GitHub action:
+https://github.com/marketplace/actions/check-version
+
+## build
+
+Checker is automatically built with pre-commit hook, but if needed, manual build is possible: </br>
+`npm run build`
+
+## usage
+
+```
+- name: Check version
+  uses: SADEInnovations/version-checker@initial
```

Comment:
The `@initial` notations can be removed from the examples as this goes to master

Reply:
Good point 👍